Makefile

```diff
@@ -86,6 +81,7 @@ INSTALL_CORE_FILES = \
     _stbt/x11.py \
     _stbt/xorg.conf.in \
     _stbt/xxhash.py \
+    stbt.conf \
```

Comment:
I see, so now we install `stbt.conf` to `/etc/stbt/stbt.conf` *and* to `$libexecdir/stbt/stbt.conf` (the latter location used to be a modified version of `stbt.conf`, now it's just the identical file).

Reply:
The ideal would be to install `/etc/stbt/stbt.conf` as identical to `/usr/.../stbt.conf` but with all the variables commented out.  This way the options and their defaults are documented and it's clear which ones are being overridden.